samples/wcfapp/Dockerfile.client

```diff
@@ -0,0 +1,8 @@
+FROM microsoft/dotnet-framework:4.7.2-sdk AS build
+WORKDIR /app
+COPY WcfClient/. ./WcfClient/
+WORKDIR /app/WcfClient
+RUN msbuild
```

Comment:
I think it would be better to illustrate a production scenario.  In that vein using `/p:Configuration=Release` feels appropriate.

Reply:
You may also choose to specify the output dir as done in https://github.com/Microsoft/dotnet-framework-docker/blob/master/samples/dotnetapp/Dockerfile#L29